modules/test/playwright/pages/configuration-admin-web/InstanceSettingsPage.ts

```diff
@@ -33,37 +33,47 @@ export class InstanceSettingsPage {
 		await this.page.getByRole('menuitem', {name: 'Export'}).click();
 	}
 
-	async goToInstanceSetting(categoryKey: string, configurationName: string) {
+	async goToInstanceSetting(
+		categoryKey: string,
+		configurationName: string,
+		nthChild?: number
+	) {
 		await this.goto();
+
 		await this.page
 			.getByRole('link', {
 				exact: true,
 				name: categoryKey,
 			})
 			.click();
-		await this.page
-			.getByRole('menuitem', {
-				exact: true,
-				name: configurationName,
-			})
-			.click();
+
+		const configurationMenuItem = this.page.getByRole('menuitem', {
+			exact: true,
+			name: configurationName,
+		});
+
+		if (nthChild === undefined) {
```

Comment:
Let's flip the order of the if else, so the first check can just be `if (nthChild) {`.

Reply:
This check will still be fine if `0` is passed since although it would be falsey calling .first() is the same as .nth(0) so it doesn't matter.